xmind/tests/test_notes_element.py

```diff
@@ -0,0 +1,109 @@
+
+        _obj1 = NotesElement()
+        _topic_mixin_element_init.assert_called_with(_obj1, None, None)
+
+        _obj2 = NotesElement(1)
+        _topic_mixin_element_init.assert_called_with(_obj2, 1, None)
+
+        _obj3 = NotesElement(1, 2)
+        _topic_mixin_element_init.assert_called_with(_obj3, 1, 2)
+
+        _obj4 = NotesElement(node=None, ownerTopic=1)
+        _topic_mixin_element_init.assert_called_with(_obj4, None, 1)
+
+        with self.assertRaises(Exception):
+            NotesElement(1, 2, 3)
+        # 4 because for last case we don't get to __init__ of super
+        self.assertEqual(_topic_mixin_element_init.call_count, 4)
+
+    def test_getContent_no_content(self):
+        _getFirstChildNodeByTagName = self._init_patch_with_name(
+            '_getFirstChildNodeByTagName', 'xmind.core.notes.NotesElement.getFirstChildNodeByTagName', return_value=False, autospec=True)
+        _topic_mixin_element_init = self._init_patch_with_name(
+            '_init_mixin', 'xmind.core.notes.TopicMixinElement.__init__', autospec=True)
+        _plain_notest_init = self._init_patch_with_name(
+            '_init_plain_notes', 'xmind.core.notes.PlainNotes.__init__', autospec=True)
+        _get_owner_topic = self._init_patch_with_name(
+            '_get_owner_topic', 'xmind.core.notes.NotesElement.getOwnerTopic', return_value='owner_topic', autospec=True)
+
+        _object = NotesElement()
+        _return_value = _object.getContent()
+        self.assertEqual(_return_value, None)
+        _getFirstChildNodeByTagName.assert_called_once_with(_object, "plain")
+        _topic_mixin_element_init.assert_called_once_with(_object, None, None)
+        _plain_notest_init.assert_not_called()
+        _get_owner_topic.assert_not_called()
+
+    def test_getContent_format_is_wrong_exception(self):
+        _content = MagicMock()
+        _getFirstChildNodeByTagName = self._init_patch_with_name(
+            '_getFirstChildNodeByTagName', 'xmind.core.notes.NotesElement.getFirstChildNodeByTagName', return_value=_content, autospec=True)
+        _topic_mixin_element_init = self._init_patch_with_name(
+            '_init_mixin', 'xmind.core.notes.TopicMixinElement.__init__', autospec=True)
+        _plain_notest_init = self._init_patch_with_name(
+            '_init_plain_notes', 'xmind.core.notes.PlainNotes.__init__', autospec=True)
+        _get_owner_topic = self._init_patch_with_name(
+            '_get_owner_topic', 'xmind.core.notes.NotesElement.getOwnerTopic', return_value='owner_topic', autospec=True)
+
+        _object = NotesElement()
+        with self.assertRaises(Exception):
+            _object.getContent("plain2")
+
+        _getFirstChildNodeByTagName.assert_called_once_with(_object, "plain2")
+        _topic_mixin_element_init.assert_called_once_with(_object, None, None)
+        _plain_notest_init.assert_not_called()
+        _get_owner_topic.assert_not_called()

```

Comment:
may we should check the exception message too? I think I saw such check in one of your tests